### PR TITLE
fix(docs): added missing package manager tab

### LINKED ISSUE #3605: docs: add more package runners
### Is your feature request related to a problem? Please describe.

Here https://nextui.org/docs/guide/cli#without-installation, there is a CLI script to install the package without global installation which uses `npx`.
What about the users using `pnpm`, `yarn`, or `bun`?

### Describe the solution you'd like

I would like to add different tabs for package runners like in this [page](https://nextui.org/docs/guide/cli#global-installation).

yarn - `yarn dlx`
pnpm - `pnpm dlx`
bun - `bun x`

### Describe alternatives you've considered

Add new tabs.

### Screenshots or Videos

Untitled

### PR DESCRIPTION
**Pull request by awesome-pro**
Closes #3605

## 📝 Description

> Add missing different package manager tabs

## ⛳️ Current behavior (updates)

Screenshot 2024-08-04 at 7 16 46 PM

Screenshot 2024-08-04 at 7 29 26 PM

## 🚀 New behavior

Screenshot 2024-08-04 at 7 14 32 PM

Screenshot 2024-08-04 at 7 17 19 PM

## Summary by CodeRabbit

- **New Features**
	- Introduced a structured `<PackageManagers>` component in documentation for easier command access across various package managers.
	- Added a new `TestTemplate` component to enhance the dropdown functionality, allowing for improved state management and selection options.

- **Documentation**
	- Updated installation and CLI usage instructions to improve clarity and organization, streamlining user experience.

### CODE REVIEW

File: apps/docs/content/docs/guide/cli.mdx [hunk 1]
```diff
@@ -30,11 +30,16 @@ To install the `CLI` globally, execute one of the following commands in your ter
 
 ### Without Installation
 
-Alternatively, you can use the `CLI` without a global installation by employing `npx`:
+Alternatively, you can use the `CLI` without a global installation by running the following commands:
```
Comment (wingkwong) [inline apps/docs/content/docs/guide/cli.mdx:33]: `commands` -> `command`

**Review by wingkwong — CHANGES_REQUESTED** [review]

**Review by wingkwong — APPROVED** [review]

**Comment by jrgarciadev** [comment]
@awesome-pro please send this PR again, branch out from `main` so we can deploy this to the docs without waiting for next version

**Comment by awesome-pro** [comment]
@jrgarciadev  I feel, you already have made a  PR, do i still need to make the PR again ?

**Comment by wingkwong** [comment]
@awesome-pro yes. Junior reverted the changes since it included some unexpected changes after changing the branch. Please branch out from main for doc changes and make a PR again.

**Pull request merged**